src/components/Prompt.tsx

```diff
@@ -119,7 +120,7 @@ export function Action({
   return (
     <Button
       variant="solid"
-      color="primary"
+      color={color}
       size={gtMobile ? 'small' : 'medium'}
```

Comment:
I think it makes sense to do this here, because `Action` is just built off of a `Button`, and we'd have access to the color on `Button` anyway.